sdk/src/ctx.rs

```diff
@@ -305,3 +306,64 @@ pub trait EigenlayerContext {
         to_block: u64,
     ) -> Result<(Vec<Address>, Vec<OperatorPubKeys>), std::io::Error>;
 }
+
+use crate::network::NetworkMultiplexer;
+use color_eyre::Result;
+use round_based::PartyIndex;
+use std::collections::BTreeMap;
+use std::sync::Arc;
+
+/// `MPCContext` trait provides access to MPC (Multi-Party Computation) functionality from the context.
+#[async_trait::async_trait]
+pub trait MPCContext {
```

Comment:
Suggest we turn `contexts` into a folder in the SDK and separate each one into its own file now. There's plenty of contexts that this would be worth doing in this PR.

Reply:
I would also suggest creating a P2P network context that provides the general p2p network setup too. It doesn't need to do anything with MPC and it would be beneficial for me to add to the docs.